src/main.rs

```diff
@@ -461,9 +494,16 @@ fn rocket() -> _ {
             .register_helper("encode-zulip-stream", Box::new(encode_zulip_stream));
     });
 
+    let rust_version = RustVersion::fetch().await.unwrap_or_default();
+    let rust_release_post = RustReleasePost::fetch().await.unwrap_or_default();
+    let teams = RustTeams::fetch().await.unwrap_or_default();
```

Comment:
This is a change in behavior, the caches will now already be prefilled with data at startup. Previously, I think the first request using a cache would actually block until some data could be retrieved.

Reply:
I think that's a good call, yeah.